src/bots/floatingMaker.ts

```diff
@@ -296,53 +331,124 @@ export class FloatingPerpMakerBot implements Bot {
-					`${this.name} cancelling order ${this.driftClient
-						.getUserAccount()!
-						.authority.toBase58()}-${o.orderId}: ${tx}`
+			try {
+				await Promise.all(
+					openOrders.map((o) => {
+						console.log(
+							`${this.name} cancelling order ${this.driftClient
+								.getUserAccount()!
+								.authority.toBase58()}-${o.orderId}`
+						);
+						return this.driftClient.cancelOrder(o.orderId);
+					})
 				);
+			} catch (e) {
+				console.log('Error canceling orders');
+				console.error(e);
+				console.log('----');
 			}
 			placeNewOrders = true;
 		}
 
 		if (placeNewOrders) {
-			const biasNum = new BN(90);
+			// The lower the bias numerator, the further below a long bid would be from the oracle price
+			// (or the further above a short ask)
+			const biasNum = new BN(this.orderOffset);
 			const biasDenom = new BN(100);
 
-			const oracleBidSpread = oracle.price.sub(vBid);
-			const tx0 = await this.driftClient.placePerpOrder({
-				marketIndex: marketIndex,
-				orderType: OrderType.LIMIT,
-				direction: PositionDirection.LONG,
-				baseAssetAmount: BASE_PRECISION.mul(new BN(1)),
-				oraclePriceOffset: oracleBidSpread
-					.mul(biasNum)
-					.div(biasDenom)
-					.neg()
-					.toNumber(), // limit bid below oracle
-			});
-			console.log(`${this.name} placing long: ${tx0}`);
-
-			const oracleAskSpread = vAsk.sub(oracle.price);
-			const tx1 = await this.driftClient.placePerpOrder({
-				marketIndex: marketIndex,
-				orderType: OrderType.LIMIT,
-				direction: PositionDirection.SHORT,
-				baseAssetAmount: BASE_PRECISION.mul(new BN(1)),
-				oraclePriceOffset: oracleAskSpread
-					.mul(biasNum)
-					.div(biasDenom)
-					.toNumber(), // limit ask above oracle
-			});
-			console.log(`${this.name} placing short: ${tx1}`);
+			try {
+				const oracleBidSpread = oracle.price.sub(vBid);
+				const oracleAskSpread = vAsk.sub(oracle.price);
+				// Altering to a vector, it'll be easier to place a set of orders later
+				const orders = [
+					{
+						marketIndex: marketIndex,
+						orderType: OrderType.LIMIT,
+						direction: PositionDirection.LONG,
+						baseAssetAmount: new BN(orderSizeLong * BASE_PRECISION.toNumber()),
+						oraclePriceOffset: oracleBidSpread
+							.mul(biasNum)
+							.div(biasDenom)
+							.neg()
+							.toNumber(), // limit bid below oracle
+					},
+					{
+						marketIndex: marketIndex,
+						orderType: OrderType.LIMIT,
+						direction: PositionDirection.SHORT,
+						baseAssetAmount: new BN(orderSizeShort * BASE_PRECISION.toNumber()),
+						oraclePriceOffset: oracleAskSpread
+							.mul(biasNum)
+							.div(biasDenom)
+							.toNumber(), // limit ask above oracle
+					},
+				];
+				await Promise.all(
+					orders.map((o) => {
+						logger.info(`${this.name} placing order: ${JSON.stringify(o)}`);
+						this.driftClient.placePerpOrder(o);
+					})
+				);
+			} catch (e) {
+				console.log('Error placing new orders');
+				console.error(e);
+				console.log('----');
```

Comment:
what error are you typically hitting that leads to this?

Reply:
Usually it seems to be "Order already processed", but I've seen the rate timeout.